config/clusters/nasa-ghg/common.values.yaml

```diff
@@ -221,7 +229,53 @@ basehub:
                   cpu_limit: 15.695
                   node_selector:
                     node.kubernetes.io/instance-type: r5.4xlarge
-
+      - display_name: SSIM GHG Workshop 2025
+        default: true
```

Comment:
This won't matter in practice because of the order but we shouldn't mark this as default or have more than one default profiles in `profileList`

Reply:
oh that's a very good catch and I missed that copying things over. Ty!

I think it's better to remove it - some people will have access to both the workshop group as well as other groups, so in that case would see two profiles. I'll remove the `default` there.